05_matplotlib.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Formative Assessment 3</p><p> </p><p>If the column "years" was longer than the column "temperaturas", would the data plot? / Si la columna "years" fuera más larga que la columna "temperaturas", ¿se trazarían los datos?</p><p> </p><p>1) Yes / Sí</p><p>2) No</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>